build.gradle

```diff
@@ -31,6 +31,8 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-viewmodel:2.6.2'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.2'
     implementation 'androidx.work:work-runtime:2.8.1'
+    implementation 'androidx.emoji2:emoji2-emojipicker:1.4.0'
+    implementation 'com.google.guava:guava:29.0-android'
```

Comment:
Btw, why is this new dependency needed? There is no apparent direct usage in any new line added

Reply:
i do not really known. by adding `implementation 'androidx.emoji2:emoji2-emojipicker:1.4.0'` we get the error:

```
/Users/bpetersen/projects/deltachat-android/src/org/thoughtcrime/securesms/connect/FetchWorker.java:12: error: cannot access ListenableFuture
public class FetchWorker extends Worker {
       ^
  class file for com.google.common.util.concurrent.ListenableFuture not found
```

additionally adding `implementation 'com.google.guava:guava:29.0-android'` fixes that for me, i found that out my some trial and error, @adbenitez would be great if you can have an additional look what is really needed

Reply:
yes without it it doesn't compiles, I was surprised because it is not required in the emojipicker tutorial, and the error is in a seemly unrelated area, but it seems to be related to this:
https://stackoverflow.com/questions/64290141/android-studio-class-file-for-com-google-common-util-concurrent-listenablefuture